packages/space-opera/src/components/model_viewer_preview/reducer.ts

```diff
@@ -91,6 +128,12 @@ export function dispatchGltfJsonString(gltfJsonString?: string) {
   return {type: SET_GLTF_JSON_STRING, payload: gltfJsonString};
 }
 
+const SET_THUMBNAILS = 'SET_THUMBNAILS'
+export async function dispatchThumbnails() {
+  const thumbnailsById = await createThumbnails();
```

Comment:
Are the thumbnails ever updated during an editor session? Or is this just called once per model?

Reply:
This function is per model load, but I'll be adding a function to append a newly-opened texture.